lte/gateway/c/session_manager/SessionState.cpp

```diff
@@ -1148,11 +1251,12 @@ bool SessionState::is_active() {
 void SessionState::set_fsm_state(
     SessionFsmState new_state, SessionStateUpdateCriteria& uc) {
   // Only log and reflect change into update criteria if the state is new
+  uint32_t local_teid_ = get_upf_local_teid();
   if (curr_state_ != new_state) {
-    MLOG(MDEBUG) << "Session " << session_id_ << " Teid " << local_teid_
-                 << " FSM state change from "
-                 << session_fsm_state_to_str(curr_state_) << " to "
-                 << session_fsm_state_to_str(new_state);
+    MLOG(MINFO) << "Session " << session_id_ << " Teid " << local_teid_
```

Comment:
Please keep it at MDEBUG level, because we are trying to keep logs less noisy.

Reply:
This is the only log prints for new session before goes to active,or  whenever state changes, it printed. So just kept as it is by adding few more fields